WooCommercePOS/WooCommercePOS/Presentation/CartView.swift

```diff
@@ -10,24 +10,43 @@ struct CartView: View {
     var body: some View {
         VStack {
             Text("Cart")
+                .frame(maxWidth: .infinity, alignment: .leading)
+                .padding(.horizontal, 32)
+                .padding(.vertical, 8)
+                .font(.title)
                 .foregroundColor(Color.white)
             ForEach(viewModel.productsInCart, id: \.product.productID) { cartProduct in
                 ProductRowView(cartProduct: cartProduct) {
                     viewModel.removeProductFromCart(cartProduct)
                 }
                 .background(Color.tertiaryBackground)
+                .padding(.horizontal, 32)
             }
             Spacer()
-            Button("Pay now") {
-                viewModel.submitCart()
-            }
-            .background(Color.secondaryBackground)
+            checkoutButton()
+                .padding(.horizontal, 32)
+
         }
         .frame(maxWidth: .infinity)
         .background(Color.secondaryBackground)
     }
+
+    @ViewBuilder
+    private func checkoutButton() -> some View {
+        Button("Checkout") {
+            viewModel.submitCart()
+        }
+        .padding(.all, 20)
+        .frame(maxWidth: .infinity, idealHeight: 120)
+        .font(.title)
+        .foregroundColor(Color.primaryBackground)
+        .background(Color.white)
+        .cornerRadius(10)
+    }
```

Comment:
> I suppose ViewBuilder is not needed.

You're right! Removed on https://github.com/woocommerce/woocommerce-ios/pull/12749/commits/8f221c4da995c459bb61b5fb5be4ff23adf96cba 

> It might be a personal preference, but I prefer having private vars for creating view components like this, so private var checkoutButton: some View instead of a `function.

Sounds good to me! Updated on: https://github.com/woocommerce/woocommerce-ios/pull/12749/commits/84e7f7f0999ab67d7ca82b461e9075323304fde9